public/app/features/explore/LogsNavigationPages.tsx

```diff
@@ -91,31 +91,31 @@ const getStyles = stylesFactory((theme: GrafanaTheme, loading: boolean) => {
       padding: 0;
       flex-direction: column;
     `,
-    page: css`
+    page: /*rgb(90,134,222) theme.colors.text.secondary*/ css`
```

Comment:
Should this comment be removed?

Reply:
Oh haha yes sorry. I put that there as reference